packages/ketcher-core/src/domain/entities/sgroupForest.ts

```diff
@@ -142,11 +142,18 @@ export class SGroupForest {
   }
 
   remove(id) {
-    assert(this.parent.has(id), 'sgid is not in the forest')
-    assert(this.children.has(id), 'sgid is not in the forest')
+    try {
+      assert(this.parent.has(id), 'sgid is not in the forest')
```

Comment:
Are you sure `assert` throws an error? I thought it just prints error into console.
Won't it be simpler
```
if (!this.parent.has(id) && !this.children.has(id)) {
  console.info('error: sgid is not in the forest')
}
```